src/Packetery/Module/ModuleHelper.php

```diff
@@ -83,9 +83,9 @@ public static function getOrderGridUrl( array $queryVars = [] ): string {
 	 *
 	 * @return bool
 	 */
-	public static function isPluginActive( string $pluginRelativePath ): bool {
-		if ( is_multisite() ) {
-			$plugins = get_site_option( 'active_sitewide_plugins' );
+	public function isPluginActive( string $pluginRelativePath ): bool {
+		if ( $this->wpAdapter->isMultisite() ) {
+			$plugins = $this->wpAdapter->getSiteOption( 'active_sitewide_plugins' );
```

Comment:
možná bych se podíval před tím isset jestli to je vůbec pole

Reply:
nechal bych na isset

Reply:
moje chyba.  špatně jsem si to přečetl.  zavírám